pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -68,6 +86,12 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			continue
 		}
 
+		if pod.Status.Phase != v1.PodPending {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPodIsNotPending
```

Comment:
For non Pending pod without PVC, should we print any error log?

Reply:
we do now, because if there is a pending pod without PVC, we will delete it and print logs whether it succeeds or not

Reply:
oh, sorry, I misunderstood...

Reply:
we can print error logs in this scenario (non-Pending pod without PVC), but this rarely happens because 

- PVC cannot be deleted if it is used by a pod
- A pod cannot run if its PVC does not exist